docs/guide/features.md

```diff
@@ -160,10 +160,11 @@ import { minify } from 'rollup-plugin-swc3'
 export default defineConfig({
   plugins: [
     minify({
+      module: true,
```

Comment:
I think it is worth to metion this line, since if it you don't add this line, the output size just about twice as `module: true`